x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/ReservedRolesStore.java

```diff
@@ -383,7 +387,11 @@ private static RoleDescriptor buildEditorRoleDescriptor() {
                 RoleDescriptor.ApplicationResourcePrivileges.builder()
                     .application("kibana-.kibana")
                     .resources("*")
-                    .privileges("all").build() },
+                    .privileges("all").build(),
+                RoleDescriptor.ApplicationResourcePrivileges.builder()
+                    .application("kibana-.*")
```

Comment:
```suggestion
                    .application("kibana-*")
```

```suggestion
                    .application("kibana-*")
```

I think this is the right syntax, given the comment in https://github.com/elastic/elasticsearch/pull/71710#discussion_r614007999 and a chat I had with @legrego 

I'll leave to Larry to comment futher.